crates/wallets/src/wallet_signer.rs

```diff
@@ -57,6 +57,53 @@ impl WalletSigner {
         Ok(Self::Local(wallet))
     }
 
+    /// Returns a list of addresses available to use with current signer
+    ///
+    /// - for Ledger and Trezor signers the number of addresses to retrieve is specified as argument
+    /// - the result for Ledger signers includes addresses available for both LedgerLive and Legacy
+    ///   derivation paths
+    /// - for Local and AWS signers the result contains a single address
+    pub async fn available_senders(&self, max: usize) -> Result<Vec<ethers_core::types::Address>> {
+        let mut senders = Vec::new();
+        match self {
+            WalletSigner::Local(local) => {
+                senders.push(local.address());
+                Ok(senders)
```

Comment:
move Ok(senders) to the bottom outside of the match

Reply:
done in https://github.com/foundry-rs/foundry/pull/7123/commits/38fc35692436fa46b8c1cf68577c913bcb339adc